vitest.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path="types/vitest.d.ts" />
```

Comment:
A combination of `<reference>` and `export {}` in the referenced definitions seemed to be required to get TypeScript to augment the appropriate interfaces when this file was imported from node_modules.